Src/VimWpf/VimHost.cs

```diff
@@ -224,6 +224,35 @@ public virtual bool IsDirty(ITextBuffer textBuffer)
             return false;
         }
 
+
+        /// <summary>
+        /// Perform the specified action when the specified text view is ready
+        /// </summary>
+        /// <param name="textView"></param>
+        /// <param name="action"></param>
+        public virtual void DoActionWhenTextViewReady(FSharpFunc<Unit, Unit> action, ITextView textView)
+        {
+            void doAction()
+            {
+                if (!textView.IsClosed)
+                {
+                    action.Invoke(null);
+                }
+            }
+            if (textView is IWpfTextView wpfTextView && !wpfTextView.VisualElement.IsLoaded)
+            {
+                // FrameworkElement.Loaded Event:
+                //
+                // Occurs when a FrameworkElement has been constructed and
+                // added to the object tree, and is ready for interaction.
+                wpfTextView.VisualElement.Loaded += (sender, e) => doAction();

```

Comment:
Think we need to unsubscribe to this event after it fires. Otherwise the delegate, which is likely to have initialization data only, will be held in memory for the lifetime of the `IWpfTextView`

Reply:
I was thinking that the vim buffer and the text view have the same lifetimes so there would not be a issue with leaking, but if we unsubscribe there is definitely no risk, so that is a better solution. I've made that change.